Example/Authenticator/View Controllers/Connection/ConnectionsViewController.swift

```diff
@@ -225,6 +225,32 @@ extension ConnectionsViewController: ConnectionCellEventsDelegate {
     }
 
     func deletePressed(id: String, showConfirmation: Bool) {
+        checkInternetConnection(id: id, showConfirmation: showConfirmation)
+    }
+
+    func reconnectPreseed(id: String) {
+        viewModel.reconnect(id: id)
+    }
+
+    func consentsPressed(id: String) {
+        viewModel.consentsPressed(connectionId: id)
+    }
+    
+    private func checkInternetConnection(id: String, showConfirmation: Bool) {
+        guard ReachabilityManager.shared.isReachable else {
+            self.showConfirmationAlert(
+                withTitle: l10n(.noInternetConnection),
+                message: l10n(.pleaseCheckAndTryAgain),
+                confirmActionTitle: l10n(.retry),
+                confirmAction: { _ in
+                    if (ReachabilityManager.shared.isReachable) {
```

Comment:
Not correct. `guard` already checks if internet connection is reachable, and if not - it shows the confirmation alert. Please, bring everything back as it was.

Reply:
As it was before, it was wrong. 
Now I have corrected it according to the task.
By clicking on retry (at a time when I do not have Internet), I should not delete the provider.
I should delete the provider by pressing retry only if I have an Internet connection.

Reply:
Ok. You are right. Make it one liner: if ReachabilityManager.shared.isReachable { self.viewModel.remove(by: id) }